composer.json

```diff
@@ -27,17 +27,18 @@
         }
     },
     "require": {
-        "php": "^5.6 || ^7.0",
+        "php": "^7.3 || ~8.0.0",
         "laminas/laminas-zendframework-bridge": "^1.0"
     },
     "require-dev": {
         "doctrine/migrations": "^1.0 || ^2.0",
         "guzzlehttp/guzzle": "^5.3.3 || ^6.3.3",
         "laminas/laminas-coding-standard": "~1.0.0",
         "laminas/laminas-loader": "^2.0",
-        "mikey179/vfsstream": "^1.6",
+        "mikey179/vfsstream": "dev-master#08a798577d677436f4f4ea3d0b3c949f64655548 as 1.6.9",
```

Comment:
To be changed before merge

Reply:
Hmm, no, only `require-dev`: I'd say "roll with it"

Reply:
Until they have a release that includes that commit... it is what it is. :)